src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -195,13 +200,22 @@ def generate_codelist_results(self):
         self.result_code_list_cols = select_codelist_cols_by_dataset_url(
             self.csvw_metadata_rdf_graph, self.dataset_url
         )
+        # Retrieving the primary key of the code list to identify the notation column
```

Comment:
Again, we can't call it the `notation` column anymore since there's another column in the CSV-W which holds the notation.